app/src/main/java/com/example/testapp/util/Util.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.testapp.util
+
+
+class Util {
```

Comment:
this should a object(static class) in kotlin for easy access from any class

Reply:
companion object

Reply:
make the class as object not adding a companion object. this class should be a static class which is object in kotlin: THis should be singleton: Please refer this